### ISSUE TITLE
How to increase the security of Expression and avoid being executed evil javascript code

### ISSUE THREAD

**Issue by WoeOm** [smartxworks/sunmao-ui#631]
**Version**

**Reproduction link**
https://sunmao-ui.com/dev.html
**Steps to reproduce**
Inside the Text component, enter `{{alert()}}`
**What is expected?**

**What is actually happening?**

**Comment by tanbowensg** [comment]
Good point. Sunmao uses `new function()` to run the code in Expression, so the expression will run globally and Sunmao can not prevent expression from running.

But there is a simple workaround to solve this problem to some extent, which is overriding some dangerous global variables(like `alert`) when evaluating expressions. Maybe we can add it into next version.

**Comment by Yuyz0112** [comment]
One thing I'm thinking about is how should we define 'dangerous code'.

Because in sunmao, the expressions were written by the app developer, the people that can responsible for the code.

**Comment by Yuyz0112** [comment]
Another point is whether the expression system cause XSS happens easier. I think the answer is yes and there is something we can improve.
